src/index.css

```
@@ -0,0 +1,39 @@
+@import "tailwindcss";
+
+@theme {
```

Stylelint (17.6.0)

[error] Unexpected unknown at-rule "@theme" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)